docs/changes.rst

```diff
@@ -218,6 +218,13 @@ Enhancements
 * Added support for retrieving the operation recorders of a connection
   via a new `operation_recorders` read-only property (Issue #976).
 
+* Extended `CIMParameter` to represent CIM parameter values in method
+  invocations. As part of that, undeprecated its `value` property and
```

Comment:
change undeprecated to removed deprecation from

Reply:
Done.